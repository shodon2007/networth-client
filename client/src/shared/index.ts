export * from './context/themeProvider'
export * from './ui'
 


