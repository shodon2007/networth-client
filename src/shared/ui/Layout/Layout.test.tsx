import {renderWithProviders} from "src/shared/tests";

describe("Testing Layout.tsx", () => {
	test("Snapshoting Layout", async () => {
		const screen = renderWithProviders();
		screen.debug();
		const layout = await screen.findByTestId("layout");

		expect(layout).toBeInTheDocument();
		expect(layout).toMatchSnapshot();
	});
});
