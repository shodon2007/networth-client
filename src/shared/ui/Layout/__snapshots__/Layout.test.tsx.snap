// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Layout.tsx Snapshoting Layout 1`] = `
<div
  class="gridLayout"
  data-testid="layout"
>
  <header
    class="header"
  >
    <div
      class="header"
    >
      <img
        class="logo"
        src=""
      />
      <div
        class="stroke"
      >
        <div
          class="strokeBg"
        >
          <img
            alt="icon"
            class="searchBarIcon"
            src=""
          />
          <input
            class="searchBar"
            placeholder="Найти . . ."
          />
        </div>
      </div>
      <button
        class="button"
      >
        dark
      </button>
      <button
        class="button"
      >
        ru
      </button>
    </div>
  </header>
  <main
    class="main"
  >
    <section
      class="feedSection"
    >
      <article
        class="article"
      >
        <header
          class="header"
        >
          <div
            class="headerMain"
          >
            <img
              alt="avatar"
              class="headerUserAvatar"
              src=""
            />
            <h1
              class="headerUserName"
            >
              ZXC Baron
            </h1>
            <span
              class="headerArticleData"
            >
              <h5
                class="headerArticleDate"
              >
                today
              </h5>
              <p
                class="headerArticleWatcher"
              >
                <img
                  alt="eye"
                  class="headerWatchIcon"
                  src=""
                />
                0
              </p>
            </span>
          </div>
          <span
            class="headerMore"
          >
             
            <button
              class="Button"
            >
              <div
                class="Button__point"
              />
              <div
                class="Button__point"
              />
              <div
                class="Button__point"
              />
            </button>
          </span>
        </header>
        <main
          class="main"
        >
          <h1
            class="mainTitle"
          >
            Great Alkil kill the bulling shit
          </h1>
          <p
            class="mainContent"
          >
            “ For every morning have other ideas. Never get some bitch for break to rest. Man you better check your last checkpoint if you’ll be thinking about to give up. Better forgot about my name ”. An ‘Alkil’ provided us such wonderful lines and enforce these fucking school boys to shut the fucking up. “
          </p>
        </main>
        <footer
          class="footer"
        >
          <header
            class="footerInteractive"
          >
            <button
              class="footerInteractiveBtn"
            >
              <h2>
                Like
              </h2>
              <p>
                0
              </p>
            </button>
            <button
              class="footerInteractiveBtn"
            >
              <h2>
                Comments
              </h2>
              <p>
                0
              </p>
            </button>
            <button
              class="footerInteractiveBtn"
            >
              <h2>
                Shared
              </h2>
              <p />
            </button>
          </header>
          <section
            class="comments"
          >
            Comments
          </section>
        </footer>
      </article>
    </section>
  </main>
  <aside
    class="aside"
  >
    <aside
      class="sidebar"
    >
      <header
        class="header"
      >
        <img
          alt="avatar"
          class="avatar"
        />
        <div
          class="header_links"
        >
          <a
            class="link"
            href="/login"
          >
            Войти
          </a>
          <a
            class="link"
            href="/login"
          >
            Зарегестрироваться
          </a>
        </div>
      </header>
      <nav
        class="navbar"
      >
        <a
          aria-current="page"
          class="navLink navLinkActive"
          href="/"
        >
          <div
            class="logo"
          >
            <div />
          </div>
          <span
            class="title"
          >
            Домой
          </span>
        </a>
        <a
          class="navLink"
          href="/profile"
        >
          <div
            class="logo"
          >
            <div />
          </div>
          <span
            class="title"
          >
            Мой профиль
          </span>
        </a>
        <a
          class="navLink"
          href="/messanger"
        >
          <div
            class="logo"
          >
            <div />
          </div>
          <span
            class="title"
          >
            Мессенджер
          </span>
        </a>
      </nav>
    </aside>
  </aside>
</div>
`;
